### ISSUE TITLE
Running Error 

### ISSUE THREAD

**Issue by xuuuluuu** [snu-mllab/Co-Mixup#4]
Hi, thanks a lot for the code.

May I check with you if you have tested the proposed method on Cifar10? How about the performance?
I am running the below script for Cifar10:

python main.py --dataset cifar10 --data_dir ./data/cifar10/ --root_dir ./experiments/cifar10 --labels_per_class 5000 --arch preactresnet18  --learning_rate 0.2 --epochs 300 --schedule 100 200 --gammas 0.1 0.1 --comix True --parallel True --m_part 20 --m_block_num 4 --mixup_alpha 2.0 --clean_lam 1.0 --m_beta 0.32 --m_gamma 1.0 --m_thres 0.83 --m_eta 0.05 --m_omega 0.001

The code encountered this error:

![image](https://user-images.githubusercontent.com/9879421/145217458-16a01af8-811a-4a18-ab6b-e1afc5c40381.png)

However, It works on Cifar100.

**Comment by Janghyun1230** [comment]
Hello, sorry for the late reply!

I could not figure out why the error occurs in your environment on CIFAR-10. 
Rather, I share my checkpoints and log files for CIFAR-10 including all the baselines (PreActResNet-18). 
: https://drive.google.com/drive/folders/1SMHJxQUUfEHD8r0W36iZSU5OxgbfkjqC?usp=sharing

Here Co-Mixup achieves 96.44% Top-1 test accuracy, whereas vanilla training achieves 95.09%.
Hope this helps! 

Best,
Jang-Hyun
